### ISSUE TITLE
Open VSX plugin outdated

### ISSUE THREAD

**Issue by niedzielski** [runem/lit-analyzer#336]
The [Open VSX release](https://open-vsx.org/extension/runem/lit-plugin) is quite old. It would be awesome if Open VSX could be updated each release just like the Visual Studio Marketplace. Thank you!

**Comment by ph08n1x** [comment]
yes I would like to bump this too, can we get it updated please? 
Really loving the plugin, great stuff!

**Comment by treeder** [comment]
Ya, the one we're getting is 4 years old... 

![image](https://github.com/runem/lit-analyzer/assets/75826/beb31e5f-3d55-4495-a8ff-fd7bd3df8760)

**Comment by treeder** [comment]
@runem thoughts on fixing this issue?

**Comment by treeder** [comment]
For reference on publishing: https://github.com/eclipse/openvsx/wiki/Publishing-Extensions

Plugin: https://open-vsx.org/extension/runem/lit-plugin

**Comment by niedzielski** [comment]
as a workaround, I think I just [downloaded from Marketplace](https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin) (look for the "Download Extension" link in the right sidebar) and installed like `codium --install-extension ~/desk/runem.lit-plugin-1.4.3.vsix`.

**Comment by ph08n1x** [comment]
Sadly that is not an option for me...

Hmmm OK at this rate, I'm thinking may be I can just publish the package on the marketplace but under a different name space as it seems we're getting no response on this ticket sadly

**Comment by treeder** [comment]
Good idea. Maybe fork this repo first so people know what they're getting.
